### ISSUE TITLE
filtering in HTML output?

### ISSUE THREAD

**Issue by juntkym** [igvteam/igv-reports#87]
Thank you for your great tool.
Is it possible to filter each column in the output HTML file?
The intended use is for rare diseases genome analysis, and I would look for causal variants from those with say, gnomad AF < 0.001 and CADD score > 30, and if none are found, I would loosen the filter to gnomad AF < 0.01 or CADD score > 20. 
I know the HTML output column allow sorting, but I thought it would be more useful if I can filter by each column.

**Comment by jrobinso** [comment]
Hi, no its not.  I would entertain a pull request for this,  it would require modifications to the html template file.   Also I would point out you can use your own template file, although admittedly there isn't any documentation on this.   Currently my suggestion would be to filter the VCF before making the report.

**Comment by barslmn** [comment]
https://github.com/user-attachments/assets/2960ebde-5be9-412c-94a6-45dee427a574

Hi, I got around implementing (with cursor) a tabulator template, with further configuration option for columns.
I added a parameter (--tabulator) for using that template. Configuration for the columns can be done with another yaml file.

https://github.com/barslmn/igv-reports/commit/505f769e83487c071a2aebd432cce9e37a72a3b1

I can open a PR if you're interested.
